src/stories/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react'
+import "./Checkbox.css"
+
+
+const Checkbox = () => {
+    const [toggle, setToggle ] = useState(false);
+    
+    const ToggleCheck = ()  => {
+        toggle ? setToggle(false) : setToggle(true)
+        console.log(toggle)
+    }
```

Comment:
```suggestion
    const ToggleCheck = ()  => setToggle(!toggle);
```

Tip: It is a good practice to try to avoid repeating yourself